minishift/openshift/auth.config.yaml

```diff
@@ -33,3 +33,5 @@ objects:
   data:
     wit.url: http://wit
     notification.serviceurl: http://f8notification
+ #  you can set "ui.url" to your locally deployed ui. by default it's localhost:8080
+ #  ui.url: http://ui
```

Comment:
I assume there's a variable with the default value set to `localhost:8080` declared in https://github.com/fabric8-services/fabric8-auth/blob/master/configuration/configuration.go ?